spec/spec_helper.rb

```diff
@@ -91,4 +98,13 @@
   #   # test failures related to randomization by passing the same `--seed` value
   #   # as the one that triggered the failure.
   #   Kernel.srand config.seed
+
+  config.before(:each) do
+    FactoryBot.rewind_sequences # Needed for sequences in Meeting, Server factory
+    Redis.new.flushdb
+  end
+
+  config.after(:each) do
+    Redis.new.flushdb
+  end
```

Comment:
There is a cleaner, better way to do this.
Will try to make use of `redis_helper` eventually